doc/languages-frameworks/gradle.section.md

```diff
@@ -65,53 +65,24 @@ path printed by `nix-build`).
 If your package can't be evaluated using a simple `pkgs.<pname>`
 expression (for example, if your package isn't located in nixpkgs, or if
 you want to override some of its attributes), you will usually have to
-pass `pkg` instead of `pname` to `gradle.fetchDeps`. There are two ways
-of doing it.
+pass `pkg` instead of `pname` to `gradle.fetchDeps`.
 
-The first is to add the derivation arguments required for getting the
-package. Using the pdftk example above:
+Use `finalAttrs.finalPackage` like so:
 
 ```nix
 { lib
 , stdenv
-# ...
-, pdftk
 }:
 
 stdenv.mkDerivation (finalAttrs: {
   # ...
   mitmCache = gradle.fetchDeps {
-    pkg = pdftk;
+    pkg = finalAttrs.finalPackage;
     data = ./deps.json;
   };
 })
 ```
 
-This allows you to `override` any arguments of the `pkg` used for
```

Comment:
While I don't strongly oppose the change, the previous way *does* allow you to override arguments using for the update script, so I think `finalPackage` should replace the second option rather than both options.